sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/TimeSeriesInsightsClientSample/ModelSettingsSamples.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Threading.Tasks;
+using static Azure.IoT.TimeSeriesInsights.Samples.SampleLogger;
+
+namespace Azure.IoT.TimeSeriesInsights.Samples
+{
+    internal class ModelSettingsSamples
+    {
+        /// <summary>
+        /// This sample demonstrates getting Time Series model settings, updating a model and changing the default type Id for a model.
+        /// </summary>
+        public async Task RunSamplesAsync(TimeSeriesInsightsClient client)
+        {
+            PrintHeader("TIME SERIES INSIGHTS MODEL SETTINGS SAMPLE");
+
+            #region Snippet:TimeSeriesInsightsSampleGetModelSettings
+            Response<TimeSeriesModelSettings> getModelSettingsResponse = await client.ModelSettings.GetAsync();
+            Console.WriteLine($"Retrieved Time Series Insights model settings:\n{JsonSerializer.Serialize(getModelSettingsResponse.Value)}");
+            #endregion
```

Comment:
endregion Snippet:TimeSeriesInsightsSampleGetModelSettings

here and everywhere below.